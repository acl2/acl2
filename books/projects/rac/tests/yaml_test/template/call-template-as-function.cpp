// RAC begin

template <int a>
int foo() {
  return a;
}

int bar() { return foo(); }

// RAC end
