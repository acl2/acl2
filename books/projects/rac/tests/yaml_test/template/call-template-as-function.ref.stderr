call-template-as-function.cpp:8 (19-24): Template function (foo) called as regular functiom
  8 | int bar() { return foo(); }
    |                    ^^^^^
note: Try instead: foo<...>(...)
